app/Console/Commands/RefreshDev.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class RefreshDev extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:refresh-dev';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Runs a development database refresh.';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        $this->call('down');
+        if (in_array(config('app.env'), ['dev']) !== true) {
+            $this->call('migrate:fresh --seeder=DevSeeder');
+        }
```

Comment:
I'm not sure I understand this step. It looks like it will clear the database only when it isn't in the dev environment. Which I think would be the opposite of what we'd want. Am I misunderstanding this?

Reply:
That was an old restriction on the scheduler. It only ran on the **dev** environment so I kept that restriction in the command steps.
I we want to to remove restriction we can and we can invert the logic to just make sure it does not run on production?

Reply:
But doesn't the code say run everywhere but 'dev'?

Reply:
I changed this to limit it from running on **testing** and **production**.

```php
        if (in_array(config('app.env'), ['testing', 'production']) !== true) {
            $this->call('migrate:fresh --seeder=DevSeeder');
            $this->call('db:seed:backup --all --restore --from=production');
        }
```